go/v1beta1/storage/elasticsearch.go

```diff
@@ -482,7 +482,174 @@ func (es *ElasticsearchStorage) CreateNote(ctx context.Context, projectId, noteI
 
 // BatchCreateNotes batch creates the specified notes in memstore.
 func (es *ElasticsearchStorage) BatchCreateNotes(ctx context.Context, pID, uID string, notes map[string]*pb.Note) ([]*pb.Note, []error) {
+	log := es.logger.Named("BatchCreateNotes")
+	log.Debug("creating notes")
+
+	var (
+		createdNotes   []*pb.Note
+		errs           []error
+		msearchReqBody bytes.Buffer
+		bulkBody       bytes.Buffer
+	)
+
+	newNotesMap := make(map[string]*pb.Note)
+
+	indexMetadataMsearch := &esMsearchQueryFragment{
+		Index: notesIndex(pID),
+	}
+
+	msearchMetadata, _ := json.Marshal(indexMetadataMsearch)
+	msearchMetadata = append(msearchMetadata, "\n"...)
+
+	for name := range notes {
+
+		noteName := fmt.Sprintf("projects/%s/notes/%s", pID, name)
+		newNotesMap[noteName] = notes[name]
+		searchBody := &esSearch{
+			Query: &filtering.Query{
+				Term: &filtering.Term{
+					"name": noteName,
+				},
+			},
+		}
+
+		data, err := json.Marshal(searchBody)
+		if err != nil {
```

Comment:
we don't really need to check for an `err` here, since we know the structure of the content we're trying to marshal.  it's just going to be a dead code path (which is why it can't be covered in tests)